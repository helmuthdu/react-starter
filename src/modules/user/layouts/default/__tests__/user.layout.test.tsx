import { render } from '@testing-library/react';
import UserLayout from '../user.layout';

const component: any = () => <p>lorem ipsum</p>;

describe('layouts -> UserLayout component', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<UserLayout>{component}</UserLayout>);

    expect(asFragment()).toMatchSnapshot();
  });
});
