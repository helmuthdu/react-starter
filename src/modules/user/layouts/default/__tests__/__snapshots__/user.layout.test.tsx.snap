// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`layouts -> UserLayout component > should match snapshot 1`] = `<DocumentFragment />`;

exports[`layouts -> UserLayout component should match snapshot 1`] = `<DocumentFragment />`;
